packages/governance/src/committee.js

```diff
@@ -60,6 +60,18 @@ const start = (zcf, privateArgs) => {
           const { voteCap } = allQuestions.get(questionHandle);
           return E(voteCap).submitVote(voterHandle, positions, 1n);
         },
+        getInvitationMaker: () =>
```

Comment:
can we match the pattern in https://github.com/Agoric/agoric-sdk/blob/f016c77e3af9b917f20b03ac4eeee1f761a53852/packages/inter-protocol/src/vaultFactory/vaultKit.js#L27-L31 ? 

I think that would mean here,
```suggestion
        invitationMakers => Far('invitation makers', {
```
And then because the offer result has to be all objects or all functions, we'd make the offer result be,
```
  voter: Far(`voter${index}`, {
  …
  invitationMakers => Far('invitation makers', {
```

Reply:
Thanks for the hint. It was close enough that I was able to get there.